src/towncrier/_git.py

```diff
@@ -10,17 +10,26 @@
 import click
 
 
-def remove_files(fragment_filenames: list[str], answer_yes: bool) -> None:
+def remove_files(
+    fragment_filenames: list[str], answer_yes: bool, answer_keep: bool
+) -> None:
```

Comment:
so, generally speaking I'm not a fan of this shell-style Python (that towncrier is full of and not specific to this PR – I'm just trying to have less of it :))

So while we're at refactoring: could we split this up into a function that does _not_ live within `_git.py` & returns a list of files that should be removed and minimizing `remove_files` to something resembling:

```python
def remove_files(files: list[str]) -> None
    if files:
        call(["git", "rm", "--quiet"] + files)
```

Reply:
sure :)

Reply:
done :)